src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/params.py

```diff
@@ -679,6 +679,26 @@ def _configure_security_policy_storage_params(cmd):
         'tenant_id': patch_arg_make_optional})
 
 
+#####
+#           sql server dns-alias
+#####
+
+
+with ParametersContext(command='sql server dns-alias') as c:
+    c.argument('server_name', arg_type=server_param_type)
+    c.argument('dns_alias_name', options_list=('--name', '-n'))
+
+    c.argument('original_server_name',
+               options_list=('--original-server'),
+               help='The name of the SQL Server to which alias is currently pointing')
+    c.argument('original_resource_group_name',
+               options_list=('--original-resource-group'),
+               help='The resource group of the SQL Server to which alias is currently pointing')
```

Comment:
"If unspecified, defaults to the targeted server's resource group". Same for original sub id